
  WARN: now based on "SDCISA-15636-Migrate-to-Java-21" branch.

--- a/Jenkinsfile
+++ b/Jenkinsfile
@@ -17,3 +17,3 @@ serviceBuildPipeline([
                         eagle:    [version: '02.01.19.00'],
-                        platform: [version: '03.06.40.01-PR-388-SNAPSHOT'],
+                        platform: [version: '${j21.platform.version}'],
                         storage:  [version: '00.24.00.00'],


--- a/blart-web/pom.xml
+++ b/blart-web/pom.xml
@@ -84,6 +84,7 @@
         <dependency>
             <groupId>io.github.java-native</groupId>
             <artifactId>jssc</artifactId>
             <version>${java-native.jssc.version}</version>
+            <scope>provided</scope>
         </dependency>
         <dependency>



