

--- a/Jenkinsfile
+++ b/Jenkinsfile
@@ -11,11 +11,11 @@ serviceBuildPipeline([
         .setBranchVersion()
-        .buildMaven([spotless: true, slim: true])
+        .buildMaven([spotless: true, ])
         .withInstances([
             instances: [
                 testInstance: [
                     services: [
                         eagle:    [version: '02.01.19.00'],
-                        platform: [version: '03.06.41.00'],
+                        platform: [version: '${j21.platform.version}'],
                         storage:  [version: '00.24.00.00'],
                         caveman:  [version: '%currentBranch%']
                     ]


--- a/caveman-web/pom.xml
+++ b/caveman-web/pom.xml
@@ -71,4 +71,5 @@
         <dependency>
             <groupId>io.github.java-native</groupId>
             <artifactId>jssc</artifactId>
+            <scope>provided</scope>
         </dependency>



