
  WARN: Now based on "SDCISA-15636-Migrate-to-Java-21" branch.

--- a/Jenkinsfile
+++ b/Jenkinsfile
@@ -20,11 +20,11 @@
                 testInstance: [
                     services: [
                         eagle:    [version: '02.01.26.01-PR-379-SNAPSHOT'],
-                        platform: [version: '03.06.40.01-PR-388-SNAPSHOT'],
+                        platform: [version: '${j21.platform.version}'],
                         storage:  [version: '00.24.00.00'],
                         deep:     [version: '%currentBranch%'],
-                        slarti:   [version: '03.15.12.01-PR-1019-SNAPSHOT'],
+                        slarti:   [version: '${j21.service.mangledVersion}'],
-                        guide:    [version: '02.14.13.01-PR-122-SNAPSHOT'],
+                        guide:    [version: '${j21.service.mangledVersion}'],
-                        trillian: [version: '03.00.38.01-PR-413-SNAPSHOT']
+                        trillian: [version: '${j21.service.mangledVersion}']
                     ]
                 ]


