

--- a/Jenkinsfile
+++ b/Jenkinsfile
@@ -13,13 +13,13 @@
     jslPipeline
             .checkout()
             .setBranchVersion()
-            .buildMaven([spotless: true])
+            .buildMaven([spotless: true])
             .withInstances([
                     instances: [
                             vehicleInstance: [
                                     services: [
                                             eagle:    [version: '02.01.26.01-PR-379-SNAPSHOT'],
-                                            platform: [version: '03.06.40.01-PR-388-SNAPSHOT'],
+                                            platform: [version: '${j21.platform.version}'],
                                             storage:  [version: '00.24.00.00'],
                                             jeltz:    [version: '%currentBranch%']
                                     ]


--- a/jeltz-process/pom.xml
+++ b/jeltz-process/pom.xml
@@ -58,4 +58,5 @@
         <dependency>
             <groupId>io.github.java-native</groupId>
             <artifactId>jssc</artifactId>
+            <scope>provided</scope>
         </dependency>


