
  WARN: now based on "SDCISA-15636-Migrate-to-Java-21" branch.

--- a/Jenkinsfile
+++ b/Jenkinsfile
@@ -10,13 +10,13 @@
     jslPipeline
         .checkout()
         .setBranchVersion()
-        .buildMaven()
+        .buildMaven()
         .withInstances([
             instances: [
                 testInstance: [
                     services: [
                         eagle:    [version: '02.01.27.01-PR-379-SNAPSHOT'],
-                        platform: [version: '03.06.46.01-PR-388-SNAPSHOT'],
+                        platform: [version: '${j21.platform.version}'],
                         storage:  [version: '00.24.00.00'],
                         loon:     [version: '%currentBranch%']
                     ]


--- a/loon-web/pom.xml
+++ b/loon-web/pom.xml
@@ -50,4 +50,5 @@
         <dependency>
             <groupId>io.github.java-native</groupId>
             <artifactId>jssc</artifactId>
+            <scope>provided</scope>
         </dependency>


