
--- a/Jenkinsfile
+++ b/Jenkinsfile
@@ -14,11 +14,11 @@ serviceBuildPipeline([
     jslPipeline
         .checkout()
         .setBranchVersion()
-        .buildMaven([slim: true])
+        .buildMaven()
         .buildDocker()
         .withInstances([instances: [
             vehicleInstance: [
                 services: [
                     eagle:    [version: '02.01.19.00'],
-                    platform: [version: '03.06.40.00'],
+                    platform: [version: '${j21.platform.version}'],
                     storage:  [version: '00.24.00.00'],

