
  WARN: now based on "SDCISA-15636-Migrate-to-Java-21" branch.


--- a/Jenkinsfile
+++ b/Jenkinsfile
@@ -10,7 +10,7 @@
     jslPipeline
             .checkout()
             .setBranchVersion()
-            .buildMaven([spotless: false])
+            .buildMaven([spotless: false])
             .buildDocker()
             .finalizeAndUpdateVersion()
             .saveArtifacts()


--- a/megacamel-web/pom.xml
+++ b/megacamel-web/pom.xml
@@ -62,4 +62,5 @@
         <dependency>
             <groupId>io.github.java-native</groupId>
             <artifactId>jssc</artifactId>
+            <scope>provided</scope>
         </dependency>



