
  WARN: Now based on "SDCISA-15636-Migrate-to-Java-21" branch.

--- a/Jenkinsfile
+++ b/Jenkinsfile
@@ -13,7 +13,7 @@
     jslPipeline
             .checkout()
             .setBranchVersion()
-            .buildMaven([spotless: true])
+            .buildMaven([spotless: true])
             .runKarma()
             .withInstances([
                     instances: [
@@ -20,6 +20,6 @@
                             backendInstance: [
                                     services: [
                                             houston:  [version: '02.01.77.01-PR-519-SNAPSHOT'],
-                                            platform: [version: '03.06.46.01-PR-388-SNAPSHOT'],
+                                            platform: [version: '${j21.platform.version}'],
                                             storage:  [version: '00.24.00.00']
                                     ]
@@ -27,8 +27,8 @@
                             vehicleInstance: [
                                     services: [
                                             eagle:    [version: '02.01.27.01-PR-379-SNAPSHOT'],
-                                            platform: [version: '03.06.46.01-PR-388-SNAPSHOT'],
+                                            platform: [version: '${j21.platform.version}'],
                                             storage:  [version: '00.24.00.00'],
-                                            slarti:   [version: '03.15.18.01-PR-1019-SNAPSHOT'],
+                                            slarti:   [version: '${j21.slarti.mangledVersion}'],
                                             minetti:  [version: '%currentBranch%']
                                     ],


