
  WARN: Now based on "SDCISA-15636-Migrate-to-Java-21" branch.

--- a/Jenkinsfile
+++ b/Jenkinsfile
@@ -13,7 +13,7 @@
     jslPipeline
             .checkout()
             .setBranchVersion()
-            .buildMaven([spotless: true])
+            .buildMaven([spotless: true])
             .runKarma()
             .withInstances([
                     instances: [
@@ -20,16 +20,16 @@
                             backendInstance: [
                                     services: [
-                                            houston:  [version: '02.01.66.01-PR-519-SNAPSHOT'],
+                                            houston:  [version: '02.01.76.00'],
-                                            platform: [version: '03.06.40.01-PR-388-SNAPSHOT'],
+                                            platform: [version: '${j21.platform.version}'],
                                             storage:  [version: '00.24.00.00']
                                     ]
                             ],
                             vehicleInstance: [
                                     services: [
-                                            eagle:    [version: '02.01.26.01-PR-379-SNAPSHOT'],
+                                            eagle:    [version: '02.01.27.00'],
-                                            platform: [version: '03.06.40.01-PR-388-SNAPSHOT'],
+                                            platform: [version: '${j21.platform.version}'],
                                             storage:  [version: '00.24.00.00'],
-                                            slarti:   [version: '03.15.09.01-PR-1019-SNAPSHOT'],
+                                            slarti:   [version: '${j21.slarti.mangledVersion}'],
                                             minetti:  [version: '%currentBranch%']
                                     ],
                                     houstonInstance: 'backendInstance'


