
  WARN: Now based on "SDCISA-15636-Migrate-to-Java-21" branch.

--- a/Jenkinsfile
+++ b/Jenkinsfile
@@ -9,7 +9,7 @@ serviceBuildPipeline([
     jslPipeline
             .checkout()
             .setBranchVersion()
-            .buildMaven()
+            .buildMaven()
             .runKarma()
             .withInstances([
                     instances: [


--- a/Jenkinsfile
+++ b/Jenkinsfile
@@ -24,11 +24,11 @@
                             testInstance: [
                                     services: [
                                             eagle:       [version: '02.01.26.01-PR-379-SNAPSHOT'],
-                                            platform:    [version: '03.06.40.01-PR-388-SNAPSHOT'],
+                                            platform:    [version: '${j21.platform.version}'],
                                             storage:     [version: '00.24.00.00'],
                                             rob:         [version: '%currentBranch%'],
-                                            slarti:      [version: '03.15.12.01-PR-1019-SNAPSHOT'],
+                                            slarti:      [version: '${j21.slarti.mangledVersion}'],
-                                            captain:     [version: '03.00.15.01-PR-103-SNAPSHOT'],
+                                            captain:     [version: '${j21.captain.mangledVersion}'],
                                             hafas:       [version: '5.41.SBB.4.15.8'],
                                             nova:        [version: '14.183.1'],
                                             'data-nova': [version: '14.183.0.20240731-1209.P.P.9238.0'],


  OBSOLETE?  --- a/Jenkinsfile
  OBSOLETE?  +++ b/Jenkinsfile
  OBSOLETE?  @@ -47,3 +47,3 @@ serviceBuildPipeline([
  OBSOLETE?               .saveArtifacts()
  OBSOLETE?  -            .executeSonar([failOnQualityGateFailure: true])
  OBSOLETE?  +            // TODO:  .executeSonar([failOnQualityGateFailure: true])
  OBSOLETE?   }


--- a/pom.xml
+++ a/pom.xml
@@ -5,7 +5,7 @@ Bcause stupid maven shoots itself in the foot ....
         <groupId>ch.post.it.paisa.service</groupId>
         <!-- Caution: when changing the parent version number, also change the
             platform.version below -->
-        <version>03.06.40.01-PR-388-SNAPSHOT</version>
+        <version>${j21.platform.version}</version>
         <relativePath/>
     </parent>
 


  OBSOLETE?  --- a/pom.xml
  OBSOLETE?  +++ a/pom.xml
  OBSOLETE?  @@ -285,6 +285,16 @@
  OBSOLETE?                   <artifactId>linti</artifactId>
  OBSOLETE?                   <version>${linti.version}</version>
  OBSOLETE?               </dependency>
  OBSOLETE?  +            <dependency>
  OBSOLETE?  +                <groupId>org.glassfish</groupId>
  OBSOLETE?  +                <artifactId>jakarta.el</artifactId>
  OBSOLETE?  +                <version>4.0.2</version> <!-- TODO move this version somewhere else -->
  OBSOLETE?  +            </dependency>
  OBSOLETE?  +            <dependency>
  OBSOLETE?  +                <groupId>org.eclipse.jetty.ee10</groupId>
  OBSOLETE?  +                <artifactId>jetty-ee10-webapp</artifactId>
  OBSOLETE?  +                <version>12.0.12</version> <!-- TODO move this version somewhere else -->
  OBSOLETE?  +            </dependency>
  OBSOLETE?           </dependencies>
  OBSOLETE?       </dependencyManagement>
  OBSOLETE?       <properties>


--- a/pom.xml
+++ a/pom.xml
@@ -328,7 +333,7 @@
          If changing one version, also change the other.
          Don't forget to change the version in the manifest.yaml, too. -->
         <linti.version>03.13.17.00</linti.version>
-        <slarti.version>03.15.10.00</slarti.version>
+        <slarti.version>${j21.slarti.mangledVersion}</slarti.version>
         <webjars.angular.sanitize.version>1.8.3</webjars.angular.sanitize.version>
     </properties>
 
     <modules>


