

--- a/Jenkinsfile
--- b/Jenkinsfile
@@ -18,7 +18,7 @@
     jslPipeline
         .checkout()
         .setBranchVersion()
-        .buildMaven([spotless: true])
+        .buildMaven([spotless: true])
         .runKarma()
         .withInstances([instances: [
             backendSel: [
@@ -51,7 +51,7 @@
             vehicleSel: [
                 services: [
                     eagle:    [version: '02.01.27.01-PR-379-SNAPSHOT'],
-                    platform: [version: '03.06.46.01-PR-388-SNAPSHOT'],
+                    platform: [version: '${j21.platform.version}'],
                     storage:  [version: '00.24.00.00'],
                     slarti:   [version: '%currentBranch%'],
                     nowwhat:  [version: '04.02.19.01-PR-123-SNAPSHOT'],
@@ -62,7 +62,7 @@
             vehicleCucumber_1: [
                 services: [
                     eagle:    [version: '02.01.27.01-PR-379-SNAPSHOT'],
-                    platform: [version: '03.06.46.01-PR-388-SNAPSHOT'],
+                    platform: [version: '${j21.platform.version}'],
                     storage:  [version: '00.24.00.00'],
                     slarti:   [version: '%currentBranch%'],
                     nowwhat:  [version: '04.02.19.01-PR-123-SNAPSHOT'],
@@ -73,7 +73,7 @@
             vehicleCucumber_take_out_service: [
                 services: [
                     eagle:    [version: '02.01.27.01-PR-379-SNAPSHOT'],
-                    platform: [version: '03.06.46.01-PR-388-SNAPSHOT'],
+                    platform: [version: '${j21.platform.version}'],
                     storage:  [version: '00.24.00.00'],
                     slarti:   [version: '%currentBranch%'],
                     nowwhat:  [version: '04.02.19.01-PR-123-SNAPSHOT'],
@@ -84,7 +84,7 @@
             st_rest: [
                 services: [
                     eagle:    [version: '02.01.27.01-PR-379-SNAPSHOT'],
-                    platform: [version: '03.06.46.01-PR-388-SNAPSHOT'],
+                    platform: [version: '${j21.platform.version}'],
                     storage:  [version: '00.24.00.00'],
                     slarti:   [version: '%currentBranch%'],
                     nowwhat:  [version: '04.02.19.01-PR-123-SNAPSHOT'],


