

--- a/Jenkinsfile
+++ b/Jenkinsfile
@@ -13,7 +13,7 @@
     jslPipeline
         .checkout()
         .setBranchVersion()
-        .buildMaven([spotless: false])
+        .buildMaven([spotless: false])
         .runKarma()
         .buildDocker()
         .withInstances([instances: [


--- a/Jenkinsfile
+++ b/Jenkinsfile
@@ -20,7 +20,7 @@
             backend: [
                 services: [
                     houston:  [version: '02.01.77.01-PR-519-SNAPSHOT'],
-                    platform: [version: '03.06.46.01-PR-388-SNAPSHOT'],
+                    platform: [version: '${j21.platform.version}'],
                     storage:  [version: '00.24.00.00'],
                     astra:    [version: '04.00.15.01-PR-168-SNAPSHOT']
                 ]


--- a/Jenkinsfile
+++ b/Jenkinsfile
@@ -28,12 +28,12 @@
             vehicle1: [
                 services: [
                     eagle:     [version: '02.01.27.01-PR-379-SNAPSHOT'],
-                    platform:  [version: '03.06.46.01-PR-388-SNAPSHOT'],
+                    platform:  [version: '${j21.platform.version}'],
                     storage:   [version: '00.24.00.00'],
                     trillian:  [version: '%currentBranch%'],
-                    slarti:    [version: '03.15.18.01-PR-1019-SNAPSHOT'],
+                    slarti:    [version: '${j21.slarti.mangledVersion}'],
                     zaphake:   [version: '03.00.22.01-PR-60-SNAPSHOT'],
-                    megacamel: [version: '03.00.15.01-PR-46-SNAPSHOT']
+                    megacamel: [version: '${j21.megacamel.mangledVersion}']
                 ],
                 houstonInstance: 'backend'
             ],


--- a/Jenkinsfile
+++ b/Jenkinsfile
@@ -40,12 +40,12 @@
             vehicle2: [
                 services: [
                     eagle:     [version: '02.01.27.01-PR-379-SNAPSHOT'],
-                    platform:  [version: '03.06.46.01-PR-388-SNAPSHOT'],
+                    platform:  [version: '${j21.platform.version}'],
                     storage:   [version: '00.24.00.00'],
                     trillian:  [version: '%currentBranch%'],
-                    slarti:    [version: '03.15.18.01-PR-1019-SNAPSHOT'],
+                    slarti:    [version: '${j21.slarti.mangledVersion}'],
                     zaphake:   [version: '03.00.22.01-PR-60-SNAPSHOT'],
-                    megacamel: [version: '03.00.15.01-PR-46-SNAPSHOT']
+                    megacamel: [version: '${j21.megacamel.mangledVersion}']
                 ],
                 houstonInstance: 'backend'
             ],


--- a/Jenkinsfile
+++ b/Jenkinsfile
@@ -52,12 +52,12 @@
             vehicle3: [
                 services: [
                     eagle:     [version: '02.01.27.01-PR-379-SNAPSHOT'],
-                    platform:  [version: '03.06.46.01-PR-388-SNAPSHOT'],
+                    platform:  [version: '${j21.platform.version}'],
                     storage:   [version: '00.24.00.00'],
                     trillian:  [version: '%currentBranch%'],
-                    slarti:    [version: '03.15.18.01-PR-1019-SNAPSHOT'],
+                    slarti:    [version: '${j21.slarti.mangledVersion}'],
                     zaphake:   [version: '03.00.22.01-PR-60-SNAPSHOT'],
-                    megacamel: [version: '03.00.15.01-PR-46-SNAPSHOT']
+                    megacamel: [version: '${j21.megacamel.mangledVersion}']
                 ],
                 houstonInstance: 'backend'
             ]


