

--- a/Jenkinsfile
+++ b/Jenkinsfile
@@ -13,7 +13,7 @@
     jslPipeline
         .checkout()
         .setBranchVersion()
-        .buildMaven([spotless: false])
+        .buildMaven([spotless: false])
         .runKarma()
         .buildDocker()
         .withInstances([instances: [
@@ -20,44 +20,44 @@
             backend: [
                 services: [
                     houston:  [version: '02.01.68.01-PR-519-SNAPSHOT'],
-                    platform: [version: '03.06.40.01-java-21-support-trillian-SNAPSHOT'],
+                    platform: [version: '${j21.platform.version}'],
                     storage:  [version: '00.24.00.00'],
-                    astra:    [version: '04.00.11.01-PR-168-SNAPSHOT']
+                    astra:    [version: '04.00.13.00']
                 ]
             ],
             vehicle1: [
                 services: [
                     eagle:     [version: '02.01.26.01-PR-379-SNAPSHOT'],
-                    platform:  [version: '03.06.40.01-java-21-support-trillian-SNAPSHOT'],
+                    platform:  [version: '${j21.platform.version}'],
                     storage:   [version: '00.24.00.00'],
                     trillian:  [version: '%currentBranch%'],
-                    slarti:    [version: '03.15.12.01-PR-1019-SNAPSHOT'],
+                    slarti:    [version: '${j21.slarti.mangledVersion}'],
                     zaphake:   [version: '03.00.22.01-PR-60-SNAPSHOT'],
-                    megacamel: [version: '03.00.15.01-PR-46-SNAPSHOT']
+                    megacamel: [version: '${j21.megacamel.mangledVersion}']
                 ],
                 houstonInstance: 'backend'
             ],
             vehicle2: [
                 services: [
                     eagle:     [version: '02.01.26.01-PR-379-SNAPSHOT'],
-                    platform:  [version: '03.06.40.01-java-21-support-trillian-SNAPSHOT'],
+                    platform:  [version: '${j21.platform.version}'],
                     storage:   [version: '00.24.00.00'],
                     trillian:  [version: '%currentBranch%'],
-                    slarti:    [version: '03.15.12.01-PR-1019-SNAPSHOT'],
+                    slarti:    [version: '${j21.slarti.mangledVersion}'],
                     zaphake:   [version: '03.00.22.01-PR-60-SNAPSHOT'],
-                    megacamel: [version: '03.00.15.01-PR-46-SNAPSHOT']
+                    megacamel: [version: '${j21.megacamel.mangledVersion}']
                 ],
                 houstonInstance: 'backend'
             ],
             vehicle3: [
                 services: [
                     eagle:     [version: '02.01.26.01-PR-379-SNAPSHOT'],
-                    platform:  [version: '03.06.40.01-java-21-support-trillian-SNAPSHOT'],
+                    platform:  [version: '${j21.platform.version}'],
                     storage:   [version: '00.24.00.00'],
                     trillian:  [version: '%currentBranch%'],
-                    slarti:    [version: '03.15.12.01-PR-1019-SNAPSHOT'],
+                    slarti:    [version: '${j21.slarti.mangledVersion}'],
                     zaphake:   [version: '03.00.22.01-PR-60-SNAPSHOT'],
-                    megacamel: [version: '03.00.15.01-PR-46-SNAPSHOT']
+                    megacamel: [version: '${j21.megacamel.mangledVersion}']
                 ],
                 houstonInstance: 'backend'
             ]


