
--- a/Jenkinsfile
+++ b/Jenkinsfile
@@ -10,13 +10,13 @@
     jslPipeline
         .checkout()
         .setBranchVersion()
-        .buildMaven([spotless: true])
+        .buildMaven([spotless: true])
         .withInstances([
             instances: [
                 testInstance: [
                     services: [
                         eagle:    [version: '02.01.26.01-PR-379-SNAPSHOT'],
-                        platform: [version: '03.06.40.01-PR-388-SNAPSHOT'],
+                        platform: [version: '${j21.platform.version}'],
                         storage:  [version: '00.24.00.00'],
                         vroom:    [version: '%currentBranch%']
                     ]


--- a/vroom-web/pom.xml
+++ b/vroom-web/pom.xml
@@ -58,6 +58,7 @@
             <groupId>io.github.java-native</groupId>
             <artifactId>jssc</artifactId>
             <version>${java-native.jssc.version}</version>
+            <scope>provided</scope>
         </dependency>
         <dependency>
             <groupId>org.webjars</groupId>


